### PR TITLE
Rework atlas post

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
Reworks atlas post a bit to add medical supplies
image

## Why's this needed? 
Listening posts should have medicine (even if its just donk pockets+medibot) and atlas had none, also a little landing pad is nice and some post gear was missing e.g. syndicate hologram module

### CODE REVIEW

**Comment by Cherman0** [comment]
Are those windows indestructable? Otherwise it would not be hard to break in. It could also use a light or two on that docking pad.

**Comment by TobleroneSwordfish** [comment]
Not sure about the full medical area, most listening posts just have the donks and mayybe a medkit. Would make more sense to have a bunk area like the original I think, as this is meant to be a long term surveillance outpost.

**Comment by JORJ949** [comment]
> Are those windows indestructable? Otherwise it would not be hard to break in. It could also use a light or two on that docking pad.

Those are indestructable yes, and I have since added a light

**Comment by JORJ949** [comment]
> Not sure about the full medical area, most listening posts just have the donks and mayybe a medkit. Would make more sense to have a bunk area like the original I think, as this is meant to be a long term surveillance outpost.

If its long term, they need somewhere to shower (and eat pizza in the shower??? (idk why I did this))

File: maps/atlas.dmm [hunk 1]
```diff
@@ -17792,6 +17933,22 @@
...
+/obj/item/parts/robot_parts/leg/right/standard,
+/obj/item/parts/robot_parts/leg/left/treads,
+/obj/item/parts/robot_parts/leg/right/treads,
+/obj/storage/crate/medical{
+	name = "medical supplies crate"
+	},
+/obj/item/storage/firstaid/brain,
+/obj/item/storage/firstaid/brute,
+/obj/item/storage/firstaid/fire,
+/obj/item/storage/firstaid/oxygen,
```
Comment (TobleroneSwordfish) [inline maps/atlas.dmm:17949]: Again, this is a lot more medical supplies than any other listening post has (afaik). Try to keep them relatively standardized.

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request merged**